master/security/app-layer-policy/index.md

```diff
@@ -144,7 +138,7 @@ Notice that from here, we get direct access to the backend database.  For exampl
 
 (Piping to `python -m json.tool` nicely formats the output.)
 
-#### Single factor authorization
+#### Single-factor authentication
```

Comment:
There's another instance of this here: https://github.com/projectcalico/calico/pull/2669/files#diff-8b697d196bb8d59ad589f28c5efd64efR289

Reply:
Fixed.